aws-crt/lib/aws-crt.rb

```diff
@@ -1,89 +1,30 @@
-require 'ffi'
-require_relative 'aws-crt/platforms'
-
 module Aws
   module Crt
-    extend FFI::Library
-
-    ffi_lib [crt_bin_path(local_platform), 'libaws-crt']
+  end
+end
```

Comment:
Defining stuff that's needed before the `require_relative` calls below

Reply:
In theory we shouldn't need to do this here.  What errors are you getting?

Reply:
This diff showed up funny.
I'm doing:
```
module Aws
  module Crt
  end
end

require_relative 'aws-crt/xyz`
```

Just so I can do the shortened `module Aws::Crt` in all the files under `aws-crt/`